From d51f42e6371620f6e1dbea8acb55ad94d6cb216a Mon Sep 17 00:00:00 2001
From: hongjiang zheng <hongjiang.a.zheng@intel.com>
Date: Sun, 14 Sep 2014 08:36:43 +0800
Subject: [PATCH 143/164] atomisp2: add isp config id into 3A statistics

BZ: 223442

This patch adds the isp_config_id(which is from ISP FW to indicate it is from
which parameter) to the 3A statistics

Change-Id: I20c986ee0d83300c1ed43aad28e17ad97d3ea00b
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-8929
Signed-off-by: hongjiang zheng <hongjiang.a.zheng@intel.com>
Signed-off-by: Bin Han <bin.b.han@intel.com>
Reviewed-on: https://android.intel.com:443/260219
---
 include/linux/atomisp.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/atomisp.h b/include/linux/atomisp.h
index e80b6a2..100c97f 100644
--- a/include/linux/atomisp.h
+++ b/include/linux/atomisp.h
@@ -307,6 +307,7 @@ struct atomisp_3a_statistics {
 	struct atomisp_3a_output __user *data;
 	struct atomisp_3a_rgby_output __user *rgby_data;
 	uint32_t exp_id; /* exposure ID */
+	uint32_t isp_config_id; /* isp config ID */
 };
 
 /**
-- 
1.7.9.5

